about.Rmd

```diff
@@ -17,7 +17,7 @@ Based on [RQM](https://cbahlai.github.io/rqm-template/). Draw much inspiration a
 - Lina Tran (Physiology)
 - James Santangelo (Biology)
```

Comment:
Since we're listing departments here, should our affiliations be changed to EEB and CSB? Unless this is referring to UTM Biology?

Reply:
Done!